pkg/cmd/run.go

```diff
@@ -539,6 +540,7 @@ func init() {
 	runCmd.Flags().Bool("fallback-readonly", false, "disable modifying the fallback file. secrets can still be read from the file.")
 	runCmd.Flags().Bool("fallback-only", false, "read all secrets directly from the fallback file, without contacting Doppler. secrets will not be updated. (implies --fallback-readonly)")
 	runCmd.Flags().Bool("no-exit-on-write-failure", false, "do not exit if unable to write the fallback file")
+	runCmd.Flags().Bool("forward-signals", false, "forward signals to the child process, regardless of whether there's a TTY allocated")
```

Comment:
Should this be true by default?

Reply:
I updated the default to reflect the calculated value, and also made the flag's description a bit more clear.